### DISCUSSION TITLE
lazer launches at 60hz every time

### DISCUSSION THREAD

**Discussion by Crozh** [ppy/osu#17412] [Q&A]
This happens with the Linux version, built and appimage. 
It fixes itself when you change to windowed or borderless, though. 
I just find it a little bit annoying to do every time and wanted to know if there is something I can do to make it launch at 144hz instead of 60, while also informing about the issue.
![Screenshot from 2022-03-22 18-56-11](https://user-images.githubusercontent.com/44332979/159583028-5965cfaa-bf14-4dd4-9ef5-f4f4d5b34734.png)
![Screenshot from 2022-03-22 18-53-22](https://user-images.githubusercontent.com/44332979/159583034-67e1467b-e25a-449b-8775-fdf4ae589069.png)

Another question I have is if the implementation for beatmap fast downloads will be free after osulazer replaces stable.

**Comment by smoogipoo** [comment]
Looks like another case of https://github.com/ppy/osu/discussions/17291

**Reply by Crozh** [reply]
It seems that way. 
I realized the same way as him: feeling the game extra choppy for no reason, even if framerate was ok.

**Reply by smoogipoo** [reply]
Can you post a screenshot of your screen layout in the linked thread?

**Reply by Crozh** [reply]
What do you mean by screen layout exactly? I apologize if this seems like a dumb question; I'm not a native english speaker, so some answers or questions might be off for you.

**Reply by smoogipoo** [reply]
The issue in the linked thread is caused by having two or more monitors with different refresh rates. If that's the case for you too, then take a screenshot of the layout.

**Reply by Crozh** [reply]
Oh, so you thought I had more screens. No, unfortunately I just have one.  
I may be incorrect, but maybe it has to do with the fact that my monitor is 143.98hz instead of the standard 144.
![image](https://user-images.githubusercontent.com/44332979/159586441-19d52325-e245-4639-83b8-adecfecdd379.png)

**Reply by smoogipoo** [reply]
Aha, so this is going to be a different issue then, thanks.

**Reply by Crozh** [reply]
I did some testing recently, and I think that's the issue. I changed my RR to 120hz and it turns out, first time launched at 60hz, but then it changed to 120hz and stood there every time I launched. 
Oh and every time I launch osulazer, I do it on fullscreen. Just to add some information. 

No problem dude, I love the team being always active fixing the problems; I just wanted to help.

**Reply by Crozh** [reply]
Sorry, scratch that, now after the 7th time running at 120hz it started to use 60 again.

**Reply by smoogipoo** [reply]
Are you using Wayland or X11?

**Reply by Crozh** [reply]
I don't know what either of those are. 
I'm kinda new to Linux as an operating system. If this responds your question, I'm using the Pop OS distro to play the game. it wasn't enhanced with anything fancy, so it's gonna be the default of whatever-that-is. 
If not, show me how can I answer your question.
